cnf-certification-test/performance/suite.go

```diff
@@ -72,13 +72,20 @@ func testSchedulingPolicyInCPUPool(env *provider.TestEnvironment,
 		pids := crclient.GetPidsFromPidNamespace(pidNamespace, testContainer)
 
 		// Check for the specified priority for each processes running in that pid namespace
-		if scheduling.ProcessPidsCPUScheduling(pids, testContainer, nonCompliantContainers, schedulingType) {
+		if scheduling.ProcessPidsCPUScheduling(pids, testContainer, nonCompliantContainersMap, schedulingType) {
```

Comment:
I'd call nonCompliantContainerMap, without s after Container. What is right from English point of view?